explorer/client/src/components/Admin/Operators/Operator.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { KeyValueList } from '../../KeyValueList'
+import { OperatorShowData } from '../../../reducers/adminOperatorsShow'
+
+interface Props {
+  operatorData: OperatorShowData
```

Comment:
I think operatorData is meant to be an optional parameter because of lines 23 and 24, right?